### PR TITLE
Release candidate

### PR DESCRIPTION
**Pull request by vgenev**
Bringing latest snapshots since 8.8 into master

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -309,7 +309,7 @@ workflows:
           requires:
             - setup
             - test-unit
-            - test-coverage
+            # - test-coverage
```
Comment (rmothilal) [inline .circleci/config.yml]: this needs to be uncommented

File: package.json [hunk 2]
```diff
@@ -69,26 +69,27 @@
...
     "@types/lodash": "4.14.149",
-    "@types/node": "13.7.1",
-    "@types/sinon": "7.5.1",
+    "@types/node": "13.7.7",
+    "@types/sinon": "7.5.2",
     "@types/tape": "4.2.34",
-    "@types/uuid": "3.4.7",
-    "@typescript-eslint/eslint-plugin": "2.19.2",
-    "@typescript-eslint/parser": "2.19.2",
+    "@types/uuid": "^7.0.0",
```
Comment (rmothilal) [inline package.json]: please remove caret

File: .circleci/config.yml (hunk 1 above)
Comment (vgenev) [reply .circleci/config.yml]: for a snapshot? or it can stay?

Comment (vgenev) [reply .circleci/config.yml]: it is gone anyway

**Pull request merged**